src/features/home/api/useGetOffersAndVenuesData.ts

```diff
@@ -54,46 +66,40 @@ export const useGetOffersAndVenuesData = (modules: HomepageModule[]) => {
       return { queryKeys: [QueryKeys.HOME_MODULE, module.id], queryFn }
     }
 
-    if (isVenuesModule(module)) {
-      const queryFn = async () => {
-        const result = await fetchMultipleVenues(module.venuesParameters, position)
-        return {
-          hits: result,
-          moduleId: module.id,
-        }
-      }
-      return { queryKeys: [QueryKeys.HOME_MODULE, module.id], queryFn }
-    }
-
     return
   })
 
-  const queriesParamsWithoutUndefined = queriesParams.filter((q) => q !== undefined) as {
+  const offersQueriesWithoutUndefined = offersQueries.filter((q) => q !== undefined) as {
     queryKeys: string[]
-    queryFn: () => Promise<OffersData> | Promise<VenuesData>
+    queryFn: () => Promise<OffersData>
   }[]
 
-  const resultList = useQueries(
-    queriesParamsWithoutUndefined.map((query) => ({
+  const offersResultList = useQueries(
+    offersQueriesWithoutUndefined.map((query) => ({
       queryKey: query.queryKeys,
       queryFn: query.queryFn,
       enabled: !!netInfo.isConnected,
     }))
   )
 
   const refetchAll = useCallback(async () => {
```

Comment:
```suggestion
  const refetchAllOffers = useCallback(async () => {
```
pour être plus clair sur ce que ça fait ?